src/components/dashboard/FeedItems/FeedListItem.js

```diff
@@ -58,6 +60,15 @@ const FeedListItem = (props: FeedListItemProps) => {
     }
   }, [fireEvent, type, onItemPress, id])
 
+  const onAnimationFinished = useCallback(
+    ({ finished }) => {
+      if (finished && isAndroidNative) {
+        setShowAndroidShadow(true)
+      }
+    },
+    [setShowAndroidShadow],
+  )
+
```

Comment:
```suggestion
  const onAnimationFinished = useCallback(({ finished }) => (finished && setAnimating(false)), [setAnimating])
```

Reply:
@serdiukov-o-nordwhale  Could you please explain the advantage your realization with my variant?